### PR TITLE
Adds some funny wizard things

### PR DESCRIPTION
**Pull request by ghost**
# Document the changes in your pull request

Adds some new wizard spells, and does minor changes to existing ones.

New stuff:
Gondola Touch - turns the target into a gondola. Yes.
Void Pull - powerful spell, that pulls people around the wizard towards him, and knockdowns them if they are near to him..

Edited ones:
Shapechange - adds to available forms a gondola, clockwork marauder and a raw prophet(unlike normal ones they have only x+ray, extended vision and telepathy. No funny mansus link and blindness).
Bolt of change - Added new pool - "Demonic" - it includes possible forms: clockwork maradeur. faithless. harvester, juggernaut, and wraith(no artificiers for balance reasons). Also includes three types of heretic monsters.
Also animal pool now contains a gondola, and robotic pool now contains a swarmer.
Contract of Apprenticeship - Added new “Melee” apprentice. It starts with a Spellblade, Instant Summons and The Traps.

And now, instead of having a boring cult dagger at the wizard den, wizard now gets a random item from the pool: Cult dagger, any heretic blade variation(since wizard is a magical guy he can use them without getting paralyzed) and a rclockcult spear.

# Wiki Documentation

Add new spells and items to wizards wiki page.

# Changelog
:cl:  
rscadd: Added new wizard spells
rscdel: Removes cult dagger from wizard spawn
rscadd: Adds a random cult knife spawner at the wizard den
rscadd: More forms for shapechange and bolt of change
soundadd: adds sound for void pull 
imageadd: adds images for void pull    
/:cl:

### CODE REVIEW

File: code/modules/mob/living/simple_animal/friendly/gondola.dm [hunk 1]
```diff
@@ -25,6 +25,8 @@
 	maxHealth = 200
 	health = 200
 	del_on_death = TRUE
+	var/mob/living/carbon/human/stored_mob //could be changed
+	var/M
```
Comment (tattax) [inline code/modules/mob/living/simple_animal/friendly/gondola.dm:29]: ?
```suggestion
```

Comment (tattax) [reply code/modules/mob/living/simple_animal/friendly/gondola.dm:29]: revert yarn

Comment (ghost) [reply code/modules/mob/living/simple_animal/friendly/gondola.dm:29]: Okay, i will revert yarn, but if i will remove var/M gondola touch wouldn’t work.

**Comment by JamieD1** [comment]
@SomeguyManperson

**Review by nesisknesis — APPROVED** [review]

**Comment by ghost** [comment]
When checks

**Comment by SomeguyManperson** [comment]
This should be atomized into smaller prs containing individual features, batching together a new assortment of spells with edits to existing spells AND edits to maps makes it harder to review

File: code/modules/mob/living/simple_animal/eldritch_demons.dm [hunk 2]
```diff
@@ -60,6 +60,20 @@
...
+	desc = "An eye supported by a mass of severed limbs, it has a piercing gaze."
+	icon_state = "raw_prophet"
+	status_flags = CANPUSH
+	icon_living = "raw_prophet"
+	melee_damage_lower = 5
+	melee_damage_upper = 10
+	maxHealth = 55
+	health = 55
+	sight = SEE_MOBS|SEE_OBJS|SEE_TURFS
+	spells_to_add = list(/obj/effect/proc_holder/spell/targeted/telepathy/eldritch)
```
Comment (tattax) [inline code/modules/mob/living/simple_animal/eldritch_demons.dm]:
```suggestion
	maxHealth = 55
	health = 55
	spells_to_add = list(/obj/effect/proc_holder/spell/targeted/telepathy/eldritch)
```

File: code/modules/mob/living/simple_animal/friendly/gondola.dm (hunk 1 above)
Comment (tattax) [reply code/modules/mob/living/simple_animal/friendly/gondola.dm:29]: it would work

**Comment by ghost** [comment]
> This should be atomized into smaller prs containing individual features, batching together a new assortment of spells with edits to existing spells AND edits to maps makes it harder to review 

I dont know how to do this at this moment

**Comment by JamieD1** [comment]
Reopen when Atomised

**Comment by Marmio64** [comment]
> > This should be atomized into smaller prs containing individual features, batching together a new assortment of spells with edits to existing spells AND edits to maps makes it harder to review
> 
> I dont know how to do this at this moment

you can just copy paste specific parts of the code from this branch into new branches.
EX: copy all spell addition code to another branch, PR it. Then spell edit code to another, and then map changes to another.

**Comment by ghost** [comment]
Sex

**Pull request closed without merging**